code/modules/events/cas_supply_increase.dm

```diff
@@ -0,0 +1,26 @@
+/datum/round_event_control/cas_supply_increase
+	name = "Dropship Point Increase"
+	typepath = /datum/round_event/cas_supply_increase
+	weight = 20
+	earliest_start = 35 MINUTES
+	alert_observers = FALSE
+	max_occurrences = 10
+
+	gamemode_blacklist = list("Crash","Civil War","Combat Patrol","Sensor Capture")
+
+/datum/round_event_control/cas_supply_increase/can_spawn_event(players_amt, gamemode)
+	if(length(GLOB.alive_human_list_faction[FACTION_TERRAGOV]) < 25)
+		return FALSE
+	if(SSpoints.dropship_points >= 250)
+		return FALSE
+	return ..()
+
+/datum/round_event/cas_supply_increase/start()
+	var/cas_points_to_be_added //var to keep track of how many point we're adding to dropship
+	for(var/mob/living/carbon/human/H in GLOB.alive_human_list_faction[FACTION_TERRAGOV])
```

Comment:
Is it intentional that a PO being temporarily dead means you receive less CAS points?

Reply:
Yes, this is intentional. The marines shouldn't be rewarded for Tad suicide missions (which is really the only thing short of a LZ breach that would kill a PO).